src/nav/apm.yml

```diff
@@ -1349,14 +1349,10 @@ pages:
             path: /docs/apm/apm-ui-pages/events/thread-profiler-tool
       - title: Error analytics
         pages:
-          - title: Errors page
+          - title: Errors 
+            path: /docs/errors-inbox/apm-tab
+          - title: Legacy errors
```

Comment:
am i understanding we relabeled this doc to 'Legacy errors'? Did we add content to that doc explaining why it's legacy and giving context and such?